mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -253,8 +255,22 @@ private boolean createInstanceActionExists(JobExecution jobExecution) {
       Action.CREATE);
   }
 
+  private boolean createAuthorityActionExists(JobExecution jobExecution) {
```

Comment:
this method returns boolean, I think there should be better to name this method smth like "isCreateAuthorityActionExists". What do you think?

Reply:
Agree. Thank you for this comment. I just did it as an analogy for already existing method "createInstanceActionExists". Yes, I will rename all of them.